server/server.go

```diff
@@ -5,8 +5,11 @@ import (
 	"sync"
 	"time"
 
+	"github.com/gogo/googleapis/google/rpc"
+	"github.com/gogo/protobuf/proto"
 	"github.com/gogo/protobuf/types"
-	"google.golang.org/genproto/googleapis/rpc/errdetails"
+	pbAny "github.com/golang/protobuf/ptypes/any"
```

Comment:
Are we still planning to get rid of these pbAny and pbStatus types and replace them with gogo types in future?